pkgs/by-name/mf/mfaktc/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenv,
+  cudatoolkit,
+  autoAddDriverRunpath,
+  cudaPackages,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mfaktc";
+  version = "0.23.4";
+
+  src = fetchFromGitHub {
+    owner = "primesearch";
+    repo = "mfaktc";
+    tag = "${finalAttrs.version}";
+    fetchSubmodules = true;
+    hash = "sha256-BlCAKzRFYPv4SYSBhNd+9yXw1PVNGkbqn2lsNeJ526A=";
+  };
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [
+    cudatoolkit
+    cudaPackages.cuda_cudart
+    autoAddDriverRunpath
+  ];
+
+  sourceRoot = "${finalAttrs.src.name}/src";
+
+  preBuild = ''
+    chmod +w ..
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    install -Dm755 ../mfaktc $out/bin
```

Ah, another one
```suggestion
    install -Dm755 ../mfaktc -t $out/bin
```
And i'm done :sweat_smile: